src/Relude/Applicative.hs

```diff
@@ -30,6 +32,17 @@ import Control.Applicative (Alternative (..), Applicative (..), Const (..), ZipL
 pass :: Applicative f => f ()
 pass = pure ()
 
+{- | For chaining applicative operations in forward applications using '(&)'
+
+>>> Just (+ 1) & appliedTo (Just 2)
+Just 3
```

Comment:
Let's also add one more example with two usages of `appliedTo` function so people can see how to change multiple functions.